README.md

```diff
@@ -2678,77 +2678,120 @@ It’s not exhaustive, as titles and details vary.
+        2015\.
+      - David Weekly, GigaOm,
+        [*5 Mistakes You Can’t Afford to Make with Stock Options*](https://gigaom.com/2011/06/05/5-mistakes-you-cant-afford-to-make-with-stock-options/),
+        2011\.
+    - **Types of equity compensation**
+     - Jeron Paul, Capshare,
+      [*RSUs vs. Options: Why RSUs (Restricted Stock Units) Could be Better Than Stock Options At Your Private Company*](https://www.capshare.com/blog/rsus-vs-options/),
+      2016\.
+     - Andy Rachleff, Wealthfront,
+      [*How Do Stock Options and RSUs Differ?*](https://blog.wealthfront.com/stock-options-versus-rsu/), 2014\.
+     - Joe Wallin, The Startup Law Blog,
+      [*Incentive Stock Options vs. Nonqualified Stock Options*](http://www.startuplawblog.com/2013/05/15/incentive-stock-options-vs-nonqualified-stock-options/), 2013\.
+     - Joe Wallin,
+      [*RSUs vs. Restricted Stock vs. Stock Options*](http://joewallin.com/2014/09/13/rsus-vs-restricted-stock-vs-stock-options/), 2014\.
+    - **Taxes**
+      - Mark P. Cussen, Investopedia,[*How Restricted Stock And RSUs Are Taxed*](http://www.investopedia.com/articles/tax/09/restricted-stock-tax.asp?performancelayout=true).
+      - Barry Kramer,
+        [🔑*The Tax Law that is (Unintentionally) Hammering Silicon Valley Employees*](https://medium.com/@barryjk/the-tax-law-that-is-unintentionally-hammering-silicon-valley-employees-894a7b54ba8a),
+        2015\.
+      - Kaye A. Thomas, Fairmark,
+        [*AMT and Long-Term Capital Gain*](http://fairmark.com/general-taxation/alternative-minimum-tax/amt-long-term-capital-gain/),
+        2014\.
+      - Robert W. Wood, Forbes,
+        [*Ten Tax Tips For Stock Options*](http://www.forbes.com/2010/03/10/10-tax-tips-stock-options-personal-finance-robert-wood.html),
+        2010\.
+      - NCEO,
+        [*Stock Options and the Alternative Minimum Tax (AMT)*](https://www.nceo.org/articles/stock-options-alternative-minimum-tax-amt)
+    - **Specialized topics**
+      - Steven Ayre, Accelerated Vesting,
+        [*What Is An 83(b) Election and When Do I Make It?*](http://acceleratedvesting.com/what-is-an-83b-election-and-when-do-i-make-it-part-1-with-graphic/),
+        2013\.
+      - Joshua Levy and Joe Wallin, The Startup Law Blog,
+        [*The Problem With Immediately Exercisable ISOs*](http://thestartuplawblog.com/the-problem-with-immediately-exercisable-isos/),
+        2015\.
+      - Tahir J. Naim, Fenwick,
+        [*Section 409A Valuations and Stock Option Grants for Start-up Technology and Life Science Companies*](http://www.fenwick.com/FenwickDocuments/409_Valuations_Stock_Options.pdf).
+      - Babak Nivi, Venture Hacks,
+        [*How to make a cap table*](http://venturehacks.com/articles/cap-table), 2007\.
+      - Mary Russell, Stock Option Counsel,
+      [*Can the Company Take Back My Vested Shares?*](http://stockoptioncounsel.com/blog/standards-ownership-canthecomanytakebackmyvestedshares), 2017\.
```

Comment:
All of these (thru Dan Shapiro's) should go under a new category "Vesting and expiration of stock options"